site/content/blog/2023-12-18-user-personas-poll/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
```

Comment:
Not the stanard social medi size .. but doe that matter?

Reply:
I'm not 100% sure but I believe most sites will resize 🤷‍♂️